tests/controls/webtest.cpp

```diff
@@ -262,8 +269,71 @@ void WebTestCase::Zoom()
 
 void WebTestCase::RunScript()
 {
-    m_browser->RunScript("document.write(\"Hello World!\");");
+#if wxUSE_WEBVIEW_IE
```

Comment:
The thing is it works when is on legacy/compatibility emulation level. It works in every case but in most cases. I don't know it if could fail. PB found some bugs with Date object and it could be more.